presto-tests/src/test/java/io/prestosql/execution/TestEventListener.java

```diff
@@ -254,12 +260,15 @@ private void assertFailedQuery(Session session, @Language("SQL") String sql, Str
         assertEquals(expectedFailure, failureInfo.getFailureMessage().orElse(null));
     }
 
-    private EventFilters buildEventFilters()
+    private EventFilters buildEventFilters(boolean ignoreSplits)
     {
-        return EventFilters.builder()
+        EventFilters.Builder builder = EventFilters.builder()
                 .setQueryCreatedFilter(event -> !event.getMetadata().getQuery().contains(IGNORE_EVENT_MARKER))
-                .setQueryCompletedFilter(event -> !event.getMetadata().getQuery().contains(IGNORE_EVENT_MARKER))
-                .build();
+                .setQueryCompletedFilter(event -> !event.getMetadata().getQuery().contains(IGNORE_EVENT_MARKER));
+        if (ignoreSplits) {
+            builder.setSplitCompletedFilter(event -> false);
+        }
+        return builder.build();
```

Comment:
You can do this fluently still
```
.setSplitCompletedFilter(event -> !ignoreSplits)
.build();
```

Reply:
this is applied, but in the next commit, should be here, in `Filter out split events from testAbortedWhileWaitingForResources`